woocommerce/woocommerce
# Prepare 2.4.0 Beta Tester Release

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Bump the version of beta tester to 2.4.0 to include the following PRs.

- https://github.com/woocommerce/woocommerce/pull/50425
- https://github.com/woocommerce/woocommerce/pull/49983
- https://github.com/woocommerce/woocommerce/pull/49966
- https://github.com/woocommerce/woocommerce/pull/49965
- https://github.com/woocommerce/woocommerce/pull/49714
- https://github.com/woocommerce/woocommerce/pull/49640
- https://github.com/woocommerce/woocommerce/pull/49341
- https://github.com/woocommerce/woocommerce/pull/49310
- https://github.com/woocommerce/woocommerce/pull/48857
- https://github.com/woocommerce/woocommerce/pull/48835




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Make sure the version is updated in all the required places.

Previous release:

-  https://github.com/woocommerce/woocommerce/pull/48806



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
